test/struct_chunk_storage.c

```diff
@@ -235,6 +235,9 @@ test_sparse_direct_chunk(hid_t fapl)
 
     TESTING("APIs for direct chunk I/O on structured chunks");
 
+    SKIPPED();
```

Comment:
Skipped since struct chunk direct I/O requires a shared chunk cache client implementation, which we don't have yet.